### PR TITLE
MCLOUD-6896: Add possibility to pass ECE-Tools environment variables …

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request (PR)  will add info on how to configure environment variables for docker containers.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-config.html

### CODE REVIEW

File: src/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -46,6 +46,32 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
 {:.bs-callout-info}
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
+```
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:54]:
```suggestion

```
```

File: src/cloud/docker/docker-config.md [hunk 2]
```diff
@@ -46,6 +46,32 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
 {:.bs-callout-info}
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:53]:
```suggestion
For example,

```

File: src/cloud/docker/docker-config.md [hunk 3]
```diff
@@ -46,6 +46,33 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
...
+
+```
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:55]:
```suggestion
```bash
```

File: src/cloud/docker/docker-config.md [hunk 4]
```diff
@@ -46,6 +46,33 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
...
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
+
+```bash
+bin/ece-docker build:compose --env-vars="{\"MAGENTO_CLOUD_VARIABLES\":{\"LOCK_PROVIDER\":\"db\",\"CRON_CONSUMERS_RUNNER\":{\"cron_run\":\"true\",\"max_messages\":5000,\"consumers\":[\"test\"]}}}"
```
Comment (hguthrie) [inline src/cloud/docker/docker-config.md:56]: This appears all on one line, which is not too horrible if you want to cut and paste.
But if you want to see what is going on, then it's more difficult.

**Review by hguthrie — APPROVED** [review]

File: src/cloud/docker/docker-config.md [hunk 5]
```diff
@@ -46,6 +46,33 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
 {:.bs-callout-info}
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:53]:
```suggestion
For example, the following command sets values for the `LOCK_PROVIDER` and `CRON_CONSUMERS_RUNNER` environment variables.
```

File: src/cloud/docker/docker-config.md (hunk 4 above)
Comment (meker12) [reply src/cloud/docker/docker-config.md:56]: I beefed up the intro sentence to show what's happening. Submitting json arrays in a command string is never pretty. ;-)

**Comment by meker12** [comment]
running tests

**Pull request merged**